src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -50,7 +60,52 @@ export class RelayFeeCalculator {
       this.capitalCostsPercent >= 0 && this.capitalCostsPercent <= 100,
       "capitalCostsPercent must be between 0 and 100 percent"
     );
+    this.capitalCostsConfig = {};
+    if (config.capitalCostsConfig) {
+      const parsed = JSON.parse(config.capitalCostsConfig);
+      for (const token of Object.keys(parsed)) {
+        this.capitalCostsConfig[token] = this.parseAndReturnCapitalCostsFromString(parsed[token]);
+      }
+    }
   }
+
+  parseAndReturnCapitalCostsFromString(capitalCosts: any): CapitalCostConfig {
```

Comment:
If you think this is a useful function, id provide it as a `static` method on the class, so a user can call it to create a validated config file from string, and so that you can strongly type the configuration parameters to the class constructor. 

id also highly suggest using `superstruct`, which is already included in the sdk. You can see an example of this being [parsed with the config store](https://github.com/across-protocol/sdk-v2/blob/master/src/contracts/acrossConfigStore.ts#L6). Superstruct will let you derive the type as well as prove to typescript an unknown type conforms to it. 

The frontend also has precendent for using superstruct, so its not a great loss to move the parsing and validation logic to the frontend constants file before instantiating your class.

one last thing, you should probably use `unknown` rather than `any`

Reply:
> so a user can call it to create a validated config file from string, and so that you can strongly type the configuration parameters to the class constructor.

Is this not strongly typed right now?